### ISSUE TITLE
Overfitting problem when using MIL Loss only and some other details

### ISSUE THREAD

**Issue by JacobKong** [sujoyp/wtalc-pytorch#3]
Hi there:

I run the code and now I have several questions:

1. When using MIL Loss only (I set the Lambda=1.0), there is an obvious overfitting problem and the highest mAP@tIoU=0.5 is 10.31. However, the paper is reported about 17.0 when  Lambda=1.0. I am wondering why. Can you share some instruction to reproduce the result?
![image](https://user-images.githubusercontent.com/10401849/56548746-7c45a080-65b3-11e9-839a-2b48d9e194e9.png)
 
2. When adding CAS Loss and training with MIL Loss, the result seems quite promising. The  mAP@tIoU=0.5 can achieve around 25.0 which is way too better than your paper. I am wondering why. Is the result reported on Tensorboard can be regarded the final result? 
![image](https://user-images.githubusercontent.com/10401849/56548990-2a514a80-65b4-11e9-8675-b833a95d51e6.png)

**Comment by JacobKong** [comment]
I am training with THUMOS14Reduced I3D feature

**Comment by sujoyp** [comment]
1. You need to change the value in threshold in detectionMAP.py Line 115 from 0.5 to something around 0.6 - 0.65 and you should be able to reproduce the results for only MILL, i.e., with Lambda of 1.0

2. The values in tensorboard should just be fine. You can also check the .log file which gets generated when during execution. You are right that the performance is a bit better in PyTorch compared to when we did with TensorFlow.

**Comment by JacobKong** [comment]
It's work! Thanks a lot!!!!!
